client/payment-details/dispute-details/dispute-footer.tsx

```diff
@@ -0,0 +1,187 @@
+/** @format **/
+
+/**
+ * External dependencies
+ */
+import React from 'react';
+import moment from 'moment';
+import classNames from 'classnames';
+import { dateI18n } from '@wordpress/date';
+import { getHistory } from '@woocommerce/navigation';
+import { __, sprintf } from '@wordpress/i18n';
+import { createInterpolateElement } from '@wordpress/element';
+import { Button, CardFooter, Flex, FlexItem } from '@wordpress/components';
+
+/**
+ * Internal dependencies
+ */
+import wcpayTracks from 'tracks';
+import type { Dispute } from 'wcpay/types/disputes';
+import { getAdminUrl } from 'wcpay/utils';
+import { formatExplicitCurrency } from 'wcpay/utils/currency';
+import { getDisputeFee, isInquiry } from 'wcpay/disputes/utils';
+import './style.scss';
+
+const formatUnixTimestamp = ( date: string, format: string ) =>
+	dateI18n( format, moment.unix( parseInt( date, 10 ) ).toISOString() );
+
+interface DisputeFooterProps {
+	dispute: Dispute;
+}
+
+const DisputeFooter: React.FC< DisputeFooterProps > = ( { dispute } ) => {
```

Comment:
Is there a clearer name for this component that helps reveal its purpose?

`PaymentDetailsDisputeStatus` is maybe too specific/long. 

- `PaymentDetails` because this is intended for display within payment details. That might be inappropriate (component should not care where it's used), but I suspect we wouldn't reuse this elsewhere.
- `DisputeStatus` because the component shows the dispute status. Maybe could be `DisputeResolveStatus`? (Unsure if this same area/component would show "you need to respond" / new status)

Reply:
Or … maybe the component is trying to serve multiple purposes!

Reply:
I've attempted with the name `DisputeResolvedMessage`, let me know what you think.

Reply:
Thanks to your patience & explanations I understand better now, and I think the deep components (each case – message + button) are `CardFooter`s. So I'm on board with `Footer` naming. 

Then it's just a question of what to name the component that chooses which footer (if any) to render, which could indeed be something like `PaymentDetailsDisputeResolutionFooter` … apologies if this is similar to where you started!

Reply:
`DisputeResolutionFooter` is where I've landed in d532d23.